lib/ohai/loader.rb

```diff
@@ -39,7 +39,13 @@ class PluginFile < Struct.new(:path, :plugin_root)
 
       # Finds all the *.rb files under the configured paths in :plugin_path
       def self.find_all_in(plugin_dir)
-        Dir[File.join(ChefConfig::PathHelper.escape_glob_dir(plugin_dir), "**", "*.rb")].map do |file|
+        if ChefConfig::PathHelper.respond_to?(:escape_glob_dir)
+          escaped = ChefConfig::PathHelper.escape_glob_dir(plugin_dir)
+        else
+          # Does not exist in 12.7 or below
```

Comment:
maybe "`escape_glob_dir` does not exist on Chef 12.7 or below" just to be extra clear?